EKF/control.cpp

```diff
@@ -119,8 +119,7 @@ void Ekf::controlFusionModes()
 	// We don't fuse flow data immediately because we have to wait for the mid integration point to fall behind the fusion time horizon.
 	// This means we stop looking for new data until the old data has been fused.
 	if (!_flow_data_ready) {
-		_flow_data_ready = _flow_buffer.pop_first_older_than(_imu_sample_delayed.time_us, &_flow_sample_delayed)
-				   && (_R_to_earth(2, 2) > _params.range_cos_max_tilt);
+		_flow_data_ready = _flow_buffer.pop_first_older_than(_imu_sample_delayed.time_us, &_flow_sample_delayed);
```

Comment:
I don't see downstream where the use of flow data when in air is stopped if the vehicle is tilted excessively other than indirectly through loss of range fusion which them after several seconds causes terrain to go invalid.

I think this change will allow use of flow data in flight when excessively tilted.

Reply:
This is here : https://github.com/PX4/ecl/pull/505/files#diff-4cac31c333867233bb8a08de1e812a9aR370